login-module/src/main/java/com/wipro/ats/bdre/security/UserRoleFetcher.java

```diff
@@ -64,10 +61,9 @@ public Users getUser(String username) {
                 users = new Users();
                 users.setUsername(jpaUser.getUsername());
                 users.setPassword(jpaUser.getPassword());
-                users.setEnabled((jpaUser.getEnabled() == true) ? (short) 1 : 0);
+                users.setEnabled((jpaUser.getEnabled()) ? (short) 1 : 0);
             }
-            //users = s.selectOne("call_procedures.GetUsers", users);
-        } catch (Exception e) {
+        } catch (BDREException e) {
             throw new RuntimeException(e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00112)